lib/Wrapper/OptSchedDDGWrapperBasic.cpp

```diff
@@ -420,35 +425,67 @@ void OptSchedDDGWrapperBasic::convertEdges(const SUnit &SU,
                                            bool IgnoreArtificialEdges) {
   const MachineInstr *instr = SU.getInstr();
   SUnit::const_succ_iterator I, E;
+#ifdef PRINT_EDGE
+  if (!IgnoreRealEdges) {
+    Logger::Info("\n\n");
+    Logger::Info("Scanning dependencies for inst (%d total, inst has %d preds)",
+                 SU.Succs.size(), SU.Preds.size());
+    SU.getInstr()->print(errs());
+  }
+#endif
+
   for (I = SU.Succs.begin(), E = SU.Succs.end(); I != E; ++I) {
     if (I->getSUnit()->isBoundaryNode())
       continue;
 
+    // errs() << "\n";
+    // I->getSUnit()->getInstr()->print(errs());
+    // Logger::Info("has latency %u", I->getLatency());
```

Comment:
Ditto

Reply:
Pretty much all of these commented out lines are debug statements I was using at one point or the other -- I'll clean them up.